controllers/dscinitialization/feature_templates.go

```diff
@@ -0,0 +1,16 @@
+package dscinitialization
+
+import (
+	"embed"
+	"path"
+)
+
+//go:embed templates
+var dsciEmbeddedFS embed.FS
+
+var (
+	baseDir      = "templates"
+	authorinoDir = path.Join(baseDir, "authorino")
+	meshDir      = path.Join(baseDir, "mesh")
```

Comment:
can we rename this folder to "servicemesh" just match whats named under kserve?

Reply:
Agreed, done 
1099d0a